functions/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "functions",
+  "description": "Cloud Functions for Firebase",
+  "scripts": {
+    "serve": "firebase emulators:start --only functions",
+    "shell": "firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "14"
```

Comment:
If you're using firebase 9 take a look at the master branch for the up to date references: https://github.com/mobinni/MealsToGo/blob/master/functions/package.json